portfolio/src/main/webapp/style.css

```diff
@@ -1,28 +1,98 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.content {
+  background-color: #E1f0C4;
+  border: 2px solid #79BEEE;
+  border-radius: 5px;
+}
+
+#intro-box {
+  background-color: #79BEEE;
+}
+
+#currently-box {
+  background-color: #B47Eb3;
+}
+
+#intro-box, #currently-box {
+  border-radius: 10px;
+  left: 35%;
+  margin-top: 20px;
+  padding: 10px;
+  position: relative;
+  width: 30%;
+}
+
+#proj-container {
+  display: flex;
+  justify-content: center;
 }
 
 #pf-image {
+  border-radius: 50%;
   display: block;
   margin-left: auto;
   margin-right: auto;
+  margin-top: 10px;
+  max-height: 200px;
+  max-width: 200px;
+  text-align: center;
   width: 50%;
 }
 
 h1, h2 {
+  color: #413C58;
   display: flex;
+  font-family: 'Raleway', sans-serif;
```

Comment:
Add the fonts.googleapis import code in all of the HTML files in case the user doesn't already have this font installed.